src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss

```diff
@@ -0,0 +1,20 @@
+.simplifiedSection{
+	.text-input-wrap{
+		height: initial;
+	}
+	.simplifiedTextbox{
+	    width: 40%;
+		float: left;
+		margin-right: 20px;
+	}
+}
+.switchToAdvanded{
```

Comment:
Advanded ?

Reply:
Change the class name to **.switchToAdvanced** . Thanks.